### ISSUE TITLE
Download of all database items required by the tourbook after an import

### ISSUE THREAD

**Issue by chrgernoe** [YacGroup/yacguide#120]
If you make a fresh installation of the app and import an existing tourbook all entries are shown with question marks.
There should be a function to download necessary database items so that all  tourbook entries are correctly shown.

**Comment by paetz** [comment]
This might become a difficult topic. Since you only have route ids, you would need to scan all routes from all sectors. This then means, you'll need to download all sectors of the whole database to check.

**Comment by chrgernoe** [comment]
You are right. Maybe a simple solution would be to store the sector ID in the tour book as well?

**Comment by paetz** [comment]
Yes, this would be the solution but how do you ensure backwards-compatibility in that case?

**Comment by chrgernoe** [comment]
I think, at the moment this not critical because we have no official release. Nevertheless, we can add a piece of code that checks that for each tour book entry a summit ID is associated. If not, just update the entry, if the database entry is available.
This code can be run with every tour book look-up in the app and should be run for the complete tour book, if it is exported. If you import an older version just ignore the missing sector ID and add the sector ID, if the database entry is available.
In general, maybe we should add a version to the exported tour book, so that possible future changes can be detected easily. Similar to the database version.
